### ISSUE TITLE
Generated Proj with '-' in name fails to start rethinkDB

### ISSUE THREAD

**Issue by casche** [rethinkdb/horizon#479]
``` bash
$ npm -v horizon
3.8.6
```

generated a 'project' via `hz init` with a dash in the name. e.g. lightbrew-horizon. 

Upon launching with `hz serve --dev`

```
App available at http://127.0.0.1:8181
RethinkDB
   ├── Admin interface: http://localhost:61329
   └── Drivers can connect to port 61328
Starting Horizon...
error: Connection to RethinkDB terminated: ReqlQueryLogicError: Database name `lightbrew-horizon` invalid (Use A-Za-z0-9_ only) in:
r(["lightbrew-horizon", "lightbrew-horizon_internal"]).forEach(function(var_0) { return r.branch(r.dbList().contains(var_0), [], r.dbCreate(var_0)); }).do(function() { return r(["collections", "users_auth", "users", "groups"]).forEach(function(var_1) { return r.branch(r.db("lightbrew-horizon_internal").tableList().contains(var_1), [], r.db("lightbrew-horizon_internal").tableCreate(var_1)); }); })
```

**Comment by deontologician** [comment]
Hey, sorry about this, we are fixing this as part of #376

**Comment by casche** [comment]
No problem @deontologician! 😄 Just thought I would give you the heads up.

**Comment by dalanmiller** [comment]
Should be fixed now with #472, thank you @casche
